docs/adguard-for-android/features/protection/private-browser.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Private browser
+sidebar_position: 7
+---
+
+:::info
+
+This article is about AdGuard for Android, a multifunctional ad blocker that protects your device at the system level. To see how it works, [download the AdGuard app](https://agrd.io/download-kb-adblock)
+
+:::
+
+In v4.7, AdGuard for Android introduced an in-app private browser. You can access it from the main screen by tapping the *Private browser* button or from the *Protection* tab, where you can find some customization options:
+
+![Private browser](https://cdn.adtidy.org/content/release_notes/ad_blocker/android/v4.7/agpb_en.png)
+
+- To set a default search engine, click *Default search engine* and choose between the available options: DuckDuckGo (default), Google, Bing, Yahoo, Baidu, or Yandex
+- To create a browser shortcut, click *Create browser widget* and follow the on-screen instructions
```

Comment:
It would be better to replace click with tap if we have already mentioned the word "tapping" earlier